design-doc-template.adoc

```diff
@@ -1,33 +1,38 @@
 ---
-# Add any category for this proposal as a yaml list, e.g.
+# Add any category for this proposal as a YAML list, e.g.
 # - core
 # - management
-# if missing, add it to _data/wildfly-categories and use its id
+# If missing, add it to _data/wildfly-categories and use its id
 categories:
 # Specify the stability level of the feature.
-# Values can be one of: experimental preview community default
+# Values can be one of: experimental, preview, community, or default
 stability-level:
 # Specify the Feature Development tracker issue for the feature.
 # This must be an issue tracked in https://github.com/orgs/wildfly/projects/7/views/1.
-# To create a Feature Development tracker issue, go to  https://github.com/wildfly/wildfly-proposals/issues/new/choose
+# To create a Feature Development tracker issue, go to https://github.com/wildfly/wildfly-proposals/issues/new/choose
 # and select 'Feature Development'
 issue:
-# Provide the github ids of the members of the feature team, organized by role.
-# Provide a single id for the 'assignee' role. Use a yaml list for the 'sme' and
+# Provide the GitHub ids of the feature team members, organized by role.
+# Provide a single id for the 'assignee' role. Use a YAML list for the 'sme' and
 # 'outside-perspective' roles, even if there is only one person in a role.
 feature-team:
  developer:
  sme:
   -
  outside-perspective:
   -
-# If this issue tracks the promotion to a higher stability level of a previously
-# completed feature, provide the URL of the https://github.com/wildfly/wildfly-proposals/issues
-# issue that was used to track the previous feature.
+# During initial development of a feature, this field should be left blank. If this issue tracks the promotion of a
```

Comment:
I think 'During initial development of a feature, this field should be left blank.' should be removed. I think you mean leave it blank in the very first, non-promotion feature analysis, but 'initial development' is unclear. I'd interpret it as meaning 'early work on the promotion that this document covers' and then wonder why I'm to leave it blank. 'If this issue tracks the promotion' covers the 'leave it blank' case.

Reply:
Updated.